cmp/chart/impl/ChartContextMenuItems.ts

```diff
@@ -0,0 +1,201 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2025 Extremely Heavy Industries Inc.
+ */
+import {isValidElement, MouseEvent, ReactNode} from 'react';
+import {cloneDeep, isEmpty, isString} from 'lodash';
+import {ChartModel} from '@xh/hoist/cmp/chart';
+import {MenuItem} from '@xh/hoist/core';
+import {Highcharts} from '@xh/hoist/kit/highcharts';
+import {Icon} from '@xh/hoist/icon';
+
+/**
+ * Highcharts supported tokens {@link https://api.highcharts.com/highcharts/exporting.buttons.contextButton.menuItems}
+ * plus Hoist's `copyToClipboard`.
+ */
+export type ChartContextMenuToken =
+    | 'viewFullscreen'
+    | 'printChart'
+    | 'downloadJPEG'
+    | 'downloadPNG'
+    | 'downloadSVG'
+    | 'downloadCSV'
+    | 'downloadXLS'
+    | 'downloadPDF'
+    | 'copyToClipboard';
+
+export interface ChartMenuItem extends Omit<MenuItem, 'actionFn' | 'items'> {
```

Comment:
Worth making the naming consistent - we could drop `Context` from the other exported members here (there's no other kind of menu), or make this `ChartContextMenuItem`